src/modules/expression/binop/range.rs

```diff
@@ -59,7 +60,11 @@ impl TranslateModule for Range {
         let from = self.from.translate(meta);
         let to = self.to.translate(meta);
         if self.neq {
-            let to_neq = translate_computation(meta, ArithOp::Sub, Some(to), Some("1".to_string()));
+            let to_neq = match &self.to.value.as_ref().unwrap() {
+                ExprType::Number(_) => (to.parse::<i64>().unwrap() - 1_i64).to_string(),
+                ExprType::VariableGet(_) => translate_computation(meta, ArithOp::Sub, Some(to), Some("1".to_string())),
+                _ => unreachable!("range expression must be either Number or VariableGet"),
```

Comment:
It's not that simple, we should be able to do this kind of things:
```ab
0..(var + 10)
```

just instead of specifing VariableGet as the only acceptable expression other than Number, allow for any other expression. Compiler checks if it has correct `Num` type, so we're safe here

Reply:
This works now:
```
hwalters@Ghostwheel ~/git/amber-lang (range-neq-opt) 
$ cat test.ab 
#!/usr/bin/env amber
let var = 5
echo 0..(var + 10)
```
```
hwalters@Ghostwheel ~/git/amber-lang (range-neq-opt) 
$ amber test.ab 
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14
```